qa/tasks/mgr/test_orchestrator_cli.py

```diff
@@ -153,3 +159,77 @@ def test_progress(self):
         evs = json.loads(self._progress_cmd('json'))['completed']
         self.assertEqual(len(evs), 1)
         self.assertIn('update_mgrs', evs[0]['message'])
+
+    def test_load_data(self):
+        data = {
+            'inventory': [
+                {
+                    'name': 'host0',
+                    'devices': [
+                        {
+                            'type': 'hdd',
+                            'id': '/dev/sda',
+                            'size': 1024**4 * 4,
+                            'rotates': True
+                        }
+                    ]
+                },
+                {
+                    'name': 'host1',
+                    'devices': [
+                        {
+                            'type': 'hdd',
+                            'id': '/dev/sda',
+                            'size': 1024**4 * 4,
+                            'rotates': True
+                        }
+                    ]
+                }
+            ],
+            'services': [
+                {
+                    'nodename': 'host0',
+                    'service_type': 'mon',
+                    'service_instance': 'a'
+                },
+                {
+                    'nodename': 'host1',
+                    'service_type': 'osd',
+                    'service_instance': '1'
+                }
+            ]
+        }
+
+        self._test_orchestrator_cmd('load_data', '-i', '-', stdin=json.dumps(data))
```

Comment:
```
2019-08-21T14:40:20.216 INFO:tasks.cephfs_test_runner:======================================================================
2019-08-21T14:40:20.217 INFO:tasks.cephfs_test_runner:ERROR: test_load_data (tasks.mgr.test_orchestrator_cli.TestOrchestratorCli)
2019-08-21T14:40:20.217 INFO:tasks.cephfs_test_runner:----------------------------------------------------------------------
2019-08-21T14:40:20.217 INFO:tasks.cephfs_test_runner:Traceback (most recent call last):
2019-08-21T14:40:20.217 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2019-08-21-1445/qa/tasks/mgr/test_orchestrator_cli.py", line 203, in test_load_data
2019-08-21T14:40:20.217 INFO:tasks.cephfs_test_runner:    self._test_orchestrator_cmd('load_data', '-i', '-', stdin=json.dumps(data))
2019-08-21T14:40:20.217 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2019-08-21-1445/qa/tasks/mgr/test_orchestrator_cli.py", line 30, in _test_orchestrator_cmd
2019-08-21T14:40:20.217 INFO:tasks.cephfs_test_runner:    return self.mgr_cluster.mon_manager.raw_cluster_cmd("test_orchestrator", *args, **kwargs)
2019-08-21T14:40:20.218 INFO:tasks.cephfs_test_runner:TypeError: raw_cluster_cmd() got an unexpected keyword argument 'stdin'
```

see http://pulpito.ceph.com/kchai-2019-08-21_12:58:47-rados-wip-kefu-testing-2019-08-21-1445-distro-basic-mira/4236663

probably you need to update `CephManager.raw_cluster_cmd()` accordingly?

Reply:
Thanks. Sorry for not spotting this, [raw_cluster_cmd in vstart_runner.py](https://github.com/ceph/ceph/blob/ea216e52f663b04c948f2d43e2736ef2bcb9b04b/qa/tasks/vstart_runner.py#L706) accept kwargs but not in normal [cephfs_test_runner](https://github.com/ceph/ceph/blob/ea216e52f663b04c948f2d43e2736ef2bcb9b04b/qa/tasks/ceph_manager.py#L1229). Update to `raw_cluster_cmd_result`.